scripts/mysql_install_db.sh

```diff
@@ -402,9 +402,9 @@ fi
 
 # Set up paths to SQL scripts required for bootstrap
 fill_help_tables="$srcpkgdatadir/fill_help_tables.sql"
-create_system_tables="$srcpkgdatadir/mysql_system_tables.sql"
+create_system_tables="$srcpkgdatadir/mariadb_system_tables.sql"
```

Comment:
there SQL filenames haven't changed (yet), included in #2392

Reply:
You are correct, it appears I had that file lying around from something previous. I will fix now.